app/javascript/pages/Ask.vue

```diff
@@ -96,7 +96,7 @@ export default {
 }
 </script>
 
-<style>
+<style scoped>
```

Comment:
what does this do, keep this style only available to this page?

Reply:
It scopes the styles just to this component and any child components.